dstyle.dd

```diff
@@ -298,6 +299,13 @@ else if (…)
 -------------------------------
     $(LI Selective imports should have a space before and after the colon (`:`) like
     `import std.range : zip`)
+    $(LI Local, selective imports should be preferred over global imports)
+    $(LI In $(I non-templated) functions should be annotated with matching attributes (`@nogc`, `@safe`, `pure`, `nothrow`))
+    $(LI $(I Templated functions) shouldn't be annotated with attributes as the compiler can infer them. However unittests
+         should be annotated to ensure function attributes (e.g. `pure nothrow @nogc @safe unittest { ... }`))
+    $(LI The use of `auto` as return type should be avoided wherever possible, as it makes the documentation easier to read))
```

Comment:
This reads like the use of `auto` makes the documentation easier to read (then why avoid it?)